packages/medusa/src/api/admin/product-options/[id]/route.ts

```diff
@@ -0,0 +1,82 @@
+import {
+  deleteProductOptionsWorkflow,
+  updateProductOptionsWorkflow,
+} from "@medusajs/core-flows"
+import {
+  AuthenticatedMedusaRequest,
+  MedusaResponse,
+  refetchEntity,
+} from "@medusajs/framework/http"
+
+import {
+  AdminGetProductOptionParamsType,
+  AdminUpdateProductOptionType,
+} from "../validators"
+import { HttpTypes } from "@medusajs/framework/types"
+import { MedusaError } from "@medusajs/framework/utils"
+
+export const GET = async (
+  req: AuthenticatedMedusaRequest<AdminGetProductOptionParamsType>,
+  res: MedusaResponse<HttpTypes.AdminProductOptionResponse>
+) => {
+  const product_option = await refetchEntity({
+    entity: "product_option",
+    idOrFilter: req.params.id,
+    scope: req.scope,
+    fields: req.queryConfig.fields,
+  })
```

Comment:
suggestion: we intend to get rid of `refetchEntitiy` entirely in favor of using the graph api, could you convert all those usage with graph instead please?